### ISSUE TITLE
Deprecate Data

### ISSUE THREAD

**Issue by fwfichtner** [dlr-eoc/ukis-pysat#170]
With STAC soon to be available for Copernicus Data Space Ecosystem I suggest to deprecate the data part in here because there is no added benefit (besides Earth Explorer, but there are also other libraries doing this better).

**Comment by MWieland** [comment]
Good idea. This makes sense in my opinion. Let's just wait until the Copernicus STAC access has been extensively tested before moving this forward.
